mpi4jax/_src/collective_ops/sendrecv.py

```diff
@@ -87,20 +88,20 @@ def sendrecv(
     if status is not None:
         status = wrap_as_hashable(status)
 
-    return tuple(
-        mpi_sendrecv_p.bind(
-            sendbuf,
-            recvbuf,
-            token,
-            source=source,
-            dest=dest,
-            sendtag=sendtag,
-            recvtag=recvtag,
-            comm=comm,
-            status=status,
-            _must_transpose=False,
-        )
+    sendbuf = ascontiguousarray(sendbuf)
```

Comment:
sendrecv unfortunately has a different signature compared to the other ops, it's first arg is not called x, so my little decorator would fail here if the user specifies the arguments as keywords. I figured the simplest solution was to just add the ascontiguousarray directly here instead of making the logic of the decorator way more complicated.
All this will be removed anyway once there is a cleaner way to deal with tokens and layouts